package.json

```diff
@@ -21,8 +21,8 @@
   "dependencies": {
     "@creativecommons/vocabulary": "1.0.0-beta.2",
     "@sentry/browser": "^5.6.0",
-    "@sentry/node": "^5.6.0",
-    "axios": "^0.18.0",
+    "@sentry/node": "^5.12.4",
```

Comment:
I'm not sure why package versions are being modified. Please revert all changes to package.json as dependency updates have the potential to be a separate issue.